app/views/pages/use-cases/twilio-video-integration.liquid

```diff
@@ -0,0 +1,302 @@
+
+Who likes a challenge? I love a good challenge but am not so fond of problems. Luckily with platformOS, challenges are made easier and problems are few:
+
+1. Retrieve the Twilio Secret
+
+    platformOS provides us with Twilio's **API Key SID** called `sid` and the **Twilio Account SID** called `auth_token`. But we must get the **API Key Secret** ourselves.
+
+2. Storing the API Keys
+
+    Once we have all the API keys, we need a place to store them securely so that we can access them later when making the connection to Twilio's video services.
+
+3. Preparing and encoding a JWT 
+
+    We needed to create a JWT encoded token that matches [Twilio's Access Token format](https://www.twilio.com/docs/iam/access-tokens) to make and persist the connection to a Twilio video `room`.
+
+## Solution
+
+Everything we needed to succeed was provided by platformOS. Our process consisted of the following five steps:
+
+<div data-autosteps></div>
+
+### Step 1: Fetch the `secret` with an API call notification 
+
+As discussed above, there are many tools to interact with an API: Curl, node.js, C#, PHP, Ruby, Python, Java and more. Below is a simple use case using Curl.
+
+#### Create API Key Secret with Curl
+Simply replace **ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX** with your Twilio `sid` and your `auth_token` from the [platformOS Partner Portal](https://portal.apps.near-me.com/): select the Instance your are working on, click Update configuration and then Edit to view the Twilio API Keys.
+<pre class="command-line" data-output="2-50" data-user="user" data-host="host"><code class="language-bash">curl -X POST https://api.twilio.com/2010-04-01/Accounts/ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX/Keys.json \ -u ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX:your_auth_token</code></pre>
+    
+We could get the API Secret using many tools but we decided to get it using platformOS. When creating a module, we wanted to do things like create rooms, recordings, compositions and more from inside the context of the module itself. 
+    
+To do this, we needed to connect directly to Twilio from platformOS.
+
+#### Create API Key Secret with platformOS - GraphQL GUI 
+Here, we created an `api_call_notification` and then used the `pos_cli gui sevre <environment>` CLI command to send and receive the response.
+
+1. **Create an API Call Notification**
+
+   To create our **API Key Secret** we send an API call with a request type of `POST` to the URL and with the headers outlined in the [Twilio Identity and Access Management documentation](https://www.twilio.com/docs/iam/keys/api-key-resource).
+
+    #### app/notifications/api_call_notification/twilio_api_secret_create.liquid
+    ```yaml
+    ---
+    name: twilio_api_secret_create
+    format: http
+    headers: |-
+    {%- assign authBasic = context.constants.TWILIO_ACCOUNT_SID | append: ":" | append: context.constants.TWILIO_ACCOUNT_AUTH_TOKEN | base64_encode -%}
+        {
+            "Content-Type": "application/json; charset=utf-8",
```

Comment:
I think charset part is not necessary :)

Reply:
Yes you right on this. I can't remember exactly why I added it, it was a while back. I think I was having issues and getting an error code from Twilio. Adding it seemed to fix things. But I just tested it and it still works so not sure what happened before. I will remove.